cloudcomposerdiff/README.md

```diff
@@ -0,0 +1,184 @@
+# A python command line tool to diff two Cloud composer environments.
+
+## Running this code
+
+### Clone repository
+
+Clone the code from GitHub to your local machine
+
+### Install locally with pip
+
+```
+pip install cloudcomposerdiff
+```
+
+### Authorizing the application using gcloud
+
+Install [gcloud](https://cloud.google.com/sdk/gcloud)
+
+Select a user which has access to both composer environments to compare.
+
+Wit this user account, initialize gcloud:
+
+```
+gcloud init
+```
+
+Next, authorize the application to access cloud composer environments:
+
+```
+gcloud auth application-default login
+```
+
+### Use the appplication
+
+```
+cloudcomposerdiff \
+--env1_project_id <> \
+--env1_location <> \
+--env1_name <> \
+--env2_project_id <> \
+--env2_location <> \
+--env2_name <>
+```
+
+## Developing this application
+
+### Clone repository
+
+Clone the code from GitHub to your local machine
+
+### Getting the correct version of python with pyenv
+
+Install [pyenv](https://github.com/pyenv/pyenv).
+
+Use pyenv to install a suitable version of python, e.g. 3.10.0
+```
+$ pyenv install 3.10.0    
+```
+
+Use pyenv to switch to python 3.10.0
+```
+$ pyenv shell 3.10.0
```

Comment:
```suggestion
pyenv shell 3.10.0
```
not important additional nit besides removing $ - Would it be worth using `local` instead of `shell`? Either works, just curious.

Reply:
Thanks for catching the "$", I have removed it

pyenv local creates a .python-version file in the current directory. 

To improve developer experience, let's avoid the creation of hidden files like this and stick to pyenv shell.